<launch>
	<arg name="marker_size" default="11.85" />
	<arg name="max_new_marker_error" default="0.08" />
	<arg name="max_track_error" default="0.2" />
        
	<arg name="cam_image_topic" default="/kinect2/hd/image_color" />
	<arg name="cam_info_topic" default="/kinect2/hd/camera_info" />        
	<arg name="output_frame" default="/kinect2_link" />
	
	

	<node ns="table" name="ar_track_alvar_bundle" pkg="ar_track_alvar" type="individualMarkersNoKinect" respawn="false" output="screen" args="$(arg marker_size) $(arg max_new_marker_error) $(arg max_track_error) $(arg cam_image_topic) $(arg cam_info_topic) $(arg output_frame) 10 5" />
</launch>
